front/lib/api/assistant/visualization.ts

```diff
@@ -176,9 +178,18 @@ if (file) {
   // for binary file:
   const arrayBuffer = await file.arrayBuffer();
 }
-\`\`\`
 
-\`fileId\` can be extracted from the \`<file id="\${FILE_ID}" type... name...>\` tags in the conversation history.
```

Comment:
This one was removed should be preserved

Reply:
Good catch!